wasp/widgets.py

```diff
@@ -221,12 +229,17 @@ def update(self):
 
 class Slider():
     """A slider to select values."""
-    def __init__(self, steps, x=10, y=90, color=0x39ff):
+    def __init__(self, steps, x=10, y=90, color=None):
         self.value = 0
         self._steps = steps
         self._stepsize = _SLIDER_TRACK / (steps-1)
         self._x = x
         self._y = y
+        # required here unfortunately,
+        # this widget has setup code related to color,
+        # so it has to be reinitialized once theme is read.
+        self._read_theme = color is not None
+        color = color or 0x39ff
```

Comment:
Better just set self._color to color here (and handle color being None in the draw method).